### ISSUE TITLE
Sketches persist between projects. 

### ISSUE THREAD

**Issue by DigitalData** [CodeDigital/CodeDigital.github.io#2]
I have no idea how to get this to stop happening. Requires tinkering to figure out. May post a question on official p5 GitHub.

**Comment by DigitalData** [comment]
Fixed in commit d3b802f (potential fix for #1 and #2)
Adding a second remove scripts ensures canvas and sketch sanitisation. This is redundant, but at this point, nothing else works.

**Comment by DigitalData** [comment]
May need to reprogram sketches to rely on create graphic function from p5.

**Comment by DigitalData** [comment]
This issue may arise again and need proper revisiting to be completely fixed, but for now I've fixed it in a very "informal" way. Technically I just brute force most of the project functions by spamming the clear() function. This is not the proper way to do this.

Another method I found to fix this later on is to name the target div (where p5 creates the sketch) with a different id for every project. This works really well and I'm happy with the results. All this does is make sure that rogue p5 code isn't able to create sketches on any pages other than its own. This brought up another issue where if loading the same page multiple times, the above issue would still persist. I haven't found a better way to deal with this other than to restrict overflow on the body. Restricting the overflow:none; in the scss body means that in project lists (which are also affected in mobile view for some reason) and p5 projects, only one created sketch can be viewed at a time.

Perlin noise generator bugs when loading music, which in turn crashes loading scripts for other projects (one issue). I just created an instance of p5 that runs in tandem with the main project sketches to load the music. This way the loadSound function is given time to load.
